src/runtime/local/kernels/Append.h

```diff
@@ -17,10 +17,10 @@
 #pragma once
 
 #include <runtime/local/context/DaphneContext.h>
+#include <runtime/local/datastructures/CSRMatrix.h>
```

Comment:
Are these header changes necessary for the clang-format update? If not I'd wrap them into a separate commit while merging (if I knew the cause for the change).

Reply:
Yes, `clang-format` sorts the includes in a way and these few files have had a few includes added after I formatted the codebase and I apparently did not `clang-format` it myself :)